File: trunk/server/net/Protocols/MsgNotify.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 飘字 协议:-801
    /// </summary>
	[Proto(value=-801,description="飘字")]
	[ProtoContract]
	public class MsgNotify
	{
        /// <summary>
        ///  消息类型
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public EMsg type;
        /// <summary>
        ///  消息
        /// </summary>
		[ProtoMember(4, IsRequired = false)]
		public string msg;
        /// <summary>
        ///  是否成功
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public bool success;
        /// <summary>
        ///  错误消息
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string info;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/notify/MsgNotify.java
/**
 * 飘字
 */
package org.ngame.protocol.notify;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import lombok.Data;
import org.ngame.domain.em.EMsg;
import org.ngame.protocol.BaseResponse;
import org.ngame.protocol.annotation.Proto;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = -801, description = "飘字")
public class MsgNotify extends BaseResponse
{

  @Protobuf(description = "消息类型", fieldType = FieldType.ENUM)
  private EMsg type;
  @Protobuf(description = "消息")
  private String msg;
}
